documentation/egress.md

```diff
@@ -15,6 +15,36 @@ Egress providers use [operations](./api/operations.md) to provide status.
 
 The `--no-http-egress` flag requires users to specify an egress provider by preventing the default HTTP response for logs, traces, dumps, gcdumps, and live metrics.
 
+## Egress Extensibility (8.0+)
+
+The `dotnet monitor` tool has transitioned to an extensible egress model. **This should not be a breaking change - by default, users migrating from `dotnet monitor` 6/7 should see no difference in `dotnet monitor`'s behavior**. 
```

Comment:
Do we want to call out the known bug https://github.com/dotnet/dotnet-monitor/issues/4427? My only concern with doing that is forgetting to update the docs once that's resolved, but technically this statement is misleading at the moment in certain situations.

Reply:
Egress extensibility didn't introduce this issue though. I don't think it should be mentioned here.